src/styles/base.js

```diff
@@ -206,7 +206,7 @@ export const modalPadding = '10px 0';
 export const modalLateralPadding = '72px';
 export const modalPortableLateralPadding = '36px';
 
-export const modalOverLayZ = 1000;
+export const modalOverLayZ = 1001;
```

Comment:
this makes sure the export modal dialog is fully visible when the tooltip is also active